src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -946,6 +946,70 @@ RppStatus rppt_slice_host(RppPtr_t srcPtr,
                                 layoutParams,
                                 rpp::deref(rppHandle));
     }
+    return RPP_SUCCESS;
+}
+
+/******************** transpose_voxel ********************/
```

Comment:
Lets not use the word voxel for these nD kernels since voxel by definition is 3D